src/mono/wasm/debugger/BrowserDebugHost/BrowserDebugHost.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <RunAnalyzers>false</RunAnalyzers>
```

Comment:
It should be true

Reply:
Making it true requires a non-trivial number of fixes. Same below.

Reply:
In that case please fill tracking issue (we ship this assembly with SDK).

Reply:
I wasn't comprehending these were shipping in the SDK.  I'll fix them.